extensions/common/context.cc

```diff
@@ -520,5 +534,19 @@ bool getAuditPolicy() {
   return shouldAudit;
 }
 
+bool sanitizeBytes(std::string* buf) {
```

Comment:
std::string&

Reply:
Using references to mutate is not consistent with the rest of this file.

Reply:
I see